### ISSUE TITLE
Customisable enable/disable of extension based on video length

### ISSUE THREAD

**Issue by Idiosyncratinom** [SkipTo/SkipTo#4]
Maybe a useful option would be to only enable this extension for a video longer than *enter time*.

Cases:
Short videos - not much need to SkipTo (less than a minute long?)
Medium videos - 1 to 5 minutes - SkipTo is quite handy. E.G. Hydraulic Press Channel has just a few key moments in their videos, but the videos are quite long.
Long videos - 5 to 10 minutes - some viewers will stay for the whole thing (SkipTo is less needed)
Longer videos - 10 to 60 minutes - again, some viewers may prefer for SkipTo bars to be absent

**Comment by SkipTo** [comment]
agreed, SkipTo works best for videos that are 1-5 minutes. An enable/disable feature that could be toggled in the options page would be a nice addition!
